README.md

```diff
@@ -28,15 +28,25 @@
 - [lbry/web-daemon](https://github.com/lbryio/web-daemon)
 - [Node](https://nodejs.org) (version >= 10)
 
-## Notes
+### Notes
 - **This repo will not run locally if you do not also have the LBRY app/daemon and the [web-daemon](https://github.com/lbryio/web-daemon) running.**
 - When running locally and completing the tipping example in Playground, the LBC donated to a creator comes from _your_ LBC balance. Otherwise, the example will fail. On production, the donated LBC comes from LBRY.
 
+### Web-daemon quickstart
+```
+git clone https://github.com/lbryio/web-daemon
+cd web-daemon
+npm i
+cp .env.sample .env
+npm run watch
+```
+
 ## Installation
+
 `npm i`
 
 ## Development
-`npm run watch` and visit [localhost:8080](http://localhost:8080)
+`npm run watch`, wait for sass bundle compilation, and visit [localhost:8080](http://localhost:8080)
```

Comment:
I don't think this addition is necessary. You can Sass getting bundled in the Terminal.